src/tooling/simulating/index.md

```diff
@@ -0,0 +1,7 @@
+# Simulating
+
+Simulating projects can be very handy. It allows users to test projects using CI, try projects without having hardware available, and many other scenarios.
+
+At the moment, there are a few ways of simulating Rust projects on Espressif chips, all of them have some limitations, but it's quickly evolving and getting better every day.
+
+In this chapter, we will discuss the different ways of simulation that are available.
```

Comment:
Given the brevity of the Wokwi and QEMU sections, maybe those two pages can just be combined into this one? If there are plans to expand on those other two sections this may not make sense, but they seem to just delegate to other documentation.

Reply:
Okay sounds good, disregard then!